kernel/data-pipeline/core/src/test/java/org/apache/shardingsphere/data/pipeline/core/util/JobDataNodeLineConvertUtilTest.java

```diff
@@ -0,0 +1,68 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.data.pipeline.core.util;
+
+import org.apache.shardingsphere.data.pipeline.api.datanode.JobDataNodeEntry;
+import org.apache.shardingsphere.data.pipeline.api.datanode.JobDataNodeLine;
+import org.apache.shardingsphere.infra.datanode.DataNode;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public final class JobDataNodeLineConvertUtilTest {
+    
+    @Test
+    public void assertConvertDataNodesToLines() {
+        Map<String, List<DataNode>> mockDataNodes = new LinkedHashMap<>();
+        List<DataNode> dataNodes = Arrays.asList(new DataNode("ds_0", "t_order_0"), new DataNode("ds_0", "t_order_1"));
+        List<DataNode> itemDataNodes = Collections.singletonList(new DataNode("ds_0", "t_order_item_0"));
+        mockDataNodes.put("t_order", dataNodes);
+        mockDataNodes.put("t_order_item", itemDataNodes);
+        List<JobDataNodeLine> jobDataNodeLines = JobDataNodeLineConvertUtil.convertDataNodesToLines(mockDataNodes);
+        assertThat(jobDataNodeLines.size(), is(1));
+        List<JobDataNodeEntry> actualNodeEntry = new ArrayList<>(jobDataNodeLines.get(0).getEntries());
+        assertThat(actualNodeEntry.get(0).getLogicTableName(), is("t_order"));
+        assertThat(actualNodeEntry.get(0).getDataNodes().size(), is(2));
+        assertThat(actualNodeEntry.get(1).getLogicTableName(), is("t_order_item"));
+        assertThat(actualNodeEntry.get(1).getDataNodes().size(), is(1));
+    }
+    
+    @Test
+    public void assertConvertDataNodesToLinesWith() {
+        Map<String, List<DataNode>> mockDataNodes = new LinkedHashMap<>();
+        List<DataNode> dataNodes = Arrays.asList(new DataNode("ds_0", "t_order_0"), new DataNode("ds_0", "t_order_1"));
+        List<DataNode> itemDataNodes = Collections.singletonList(new DataNode("ds_0", "t_order_item_0"));
+        mockDataNodes.put("t_order", dataNodes);
+        mockDataNodes.put("t_order_item", itemDataNodes);
+        List<JobDataNodeLine> jobDataNodeLines = JobDataNodeLineConvertUtil.convertDataNodesToLines(mockDataNodes);
+        assertThat(jobDataNodeLines.size(), is(1));
+        List<JobDataNodeEntry> actualNodeEntry = new ArrayList<>(jobDataNodeLines.get(0).getEntries());
+        assertThat(actualNodeEntry.get(0).getLogicTableName(), is("t_order"));
+        assertThat(actualNodeEntry.get(0).getDataNodes().size(), is(2));
+        assertThat(actualNodeEntry.get(1).getLogicTableName(), is("t_order_item"));
+        assertThat(actualNodeEntry.get(1).getDataNodes().size(), is(1));
+    }
```

Comment:
Looks the content of `assertConvertDataNodesToLinesWith` is the same as `assertConvertDataNodesToLines`